### PR TITLE
adding NetGateway Support

### PR DESCRIPTION
**Pull request by LePhanFF**

### CODE REVIEW

File: internal/pritunl/route.go [hunk 1]
```diff
@@ -41,6 +41,9 @@ func ConvertMapToRoute(data map[string]interface{}) Route {
 	if v, ok := data["nat"]; ok {
 		route.Nat = v.(bool)
 	}
+	if v, ok := data["net_gateway"]; ok {
+		route.NetGateway = v.(bool)
```
Comment (disc) [inline internal/pritunl/route.go:45]: I see the setter of the `NetGateway` property but didn't see any place where you use this property in the provider. Do you need just to store this property in the state or I miss something?

**Review by disc — CHANGES_REQUESTED** [review]

File: internal/pritunl/route.go (hunk 1 above)
Comment (LePhanFF) [reply internal/pritunl/route.go:45]: We need to create net gateway route, would this change set the route type when it invokes the api to create the route?

Comment (LePhanFF) [reply internal/pritunl/route.go:45]: I found some places where it could be used,  would these changes work, please suggest

**Comment by LePhanFF** [comment]
replaced with https://github.com/disc/terraform-provider-pritunl/pull/39

**Pull request closed without merging**